src/index.jsx

```diff
@@ -98,7 +98,11 @@ async function init() {
                           props => (
                             <ProtectedContent
                               public
-                              filter={() => store.dispatch(fetchLogin())}
+                              filter={() => {
+                                setTimeout(() => {
+                                  store.dispatch(fetchLogin());
+                                }, 10000);
+                              }}
```

Comment:
note this is a temp sleep function to let you to see the button state during the 10s sleep

it will be removed before merging